PiwikTracker/PiwikTracker.swift

```diff
@@ -38,10 +39,13 @@ final public class PiwikTracker: NSObject {
     ///   - siteId: The unique site id generated by the server when a new site was created.
     ///   - queue: The queue to use to store all analytics until it is dispatched to the server.
     ///   - dispatcher: The dispatcher to use to transmit all analytics to the server.
-    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher) {
+    required public init(siteId: String, queue: Queue, dispatcher: Dispatcher, suiteName: String? = nil) {
```

Comment:
I agree, use the combination on baseURL and siteId as key., full string or hash, sounds like a good idea.

I would probably not expose it to users of the framework.